CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+- Improved handling of GLIBC version incompatibility on Linux systems ([#23](https://github.com/stjude-rust-labs/sprocket-vscode/issues/23))
```

Comment:
Generally we've kept CHANGELOG entries to a single bullet point and let people follow the links for more context. Would you mind removing the sub-bullet points?